recipe/windows/activate.bat

```diff
@@ -68,9 +68,10 @@ if errorlevel 1 (
 
 set "CUDA_PATH=%CUDA_PATH%"
 set "CUDA_HOME=%CUDA_PATH%"
-set "CFLAGS=%CFLAGS% -I%CUDA_HOME%\include"
-set "CPPFLAGS=%CPPFLAGS% -I%CUDA_HOME%\include"
-set "CXXFLAGS=%CXXFLAGS% -I%CUDA_HOME%\include"
+
```

Comment:
Could you add echos for the old values before resetting them? The windows CI (in contrast to linux/osc) does not [show](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=250854&view=logs&j=00f5923e-fdef-5026-5091-0d5a0b3d5a2c&t=3cc4a9ed-60e1-5810-6eb3-5f9cd4a26dba&l=406) what the values were initially, and it's kinda hard to know which ones come from the config, which from various activation scripts, and which from cmake, cf.
```
cl /c /I[...] /Zi /W1 /WX- /diagnostics:classic /Od /Ob0 /D WIN32 /D _WINDOWS /D "CMAKE_INTDIR=\"Debug\"" /D _MBCS /Gm- /EHsc /RTC1 /MDd /GS /fp:precise /Qspectre /Zc:wchar_t /Zc:forScope /Zc:inline /GR /Fo"cmTC_fd0fd.dir\Debug\\" /Fd"cmTC_fd0fd.dir\Debug\vc141.pdb" /Gd /TP /errorReport:queue
```

Reply:
Nevermind, I guess this should already work with the various `C{,PP,XX}FLAGS_CONDA_NVCC_BACKUP` variables.